rama-net/src/tls/keylog.rs

```diff
@@ -0,0 +1,142 @@
+//! Keylog facility used by any tls implementation
+//! supported by rama, and which can be used for your owns as well.
+//!
+//! Center to thsi module is the `KeyLogger` which is a wrapper around
```

Comment:
```suggestion
//! Center to this module is the `KeyLogger` which is a wrapper around
```

I guess typo from myself